src/test/java/com/google/api/codegen/testdata/python_doc_types_library.baseline

```diff
@@ -0,0 +1,65 @@
+============== file: google/cloud/proto/example/library_v1/types.py ==============
```

Comment:
> @lukesneeringer how does that sound to you? Simply changing the semantics of the package name to be of the form google.cloud.example_v1 and having the gapic and proto paths just added to that package name.
  
That is fine by me. It might even be ideal by me. :-)